sympy/assumptions/ask_generated.py

```diff
@@ -97,168 +97,168 @@ def get_known_facts_dict():
 
     """
     return {
-        Q.algebraic: (set([Q.algebraic, Q.commutative, Q.complex, Q.finite]),
-        set([Q.infinite, Q.negative_infinite, Q.positive_infinite,
-        Q.transcendental])),
-        Q.antihermitian: (set([Q.antihermitian]), set([])),
-        Q.commutative: (set([Q.commutative]), set([])),
-        Q.complex: (set([Q.commutative, Q.complex, Q.finite]),
-        set([Q.infinite, Q.negative_infinite, Q.positive_infinite])),
-        Q.complex_elements: (set([Q.complex_elements]), set([])),
-        Q.composite: (set([Q.algebraic, Q.commutative, Q.complex, Q.composite,
+        Q.algebraic: ({Q.algebraic, Q.commutative, Q.complex, Q.finite},
+        {Q.infinite, Q.negative_infinite, Q.positive_infinite,
```

Comment:
This file is automatically generated and should not be modified without modifying the script that generates it.

Reply:
Ah, I'll fix that. @oscarbenjamin Any other autogenerated files I should be aware of?